internal/rbd/nodeserver.go

```diff
@@ -149,6 +149,19 @@ func healerStageTransaction(ctx context.Context, cr *util.Credentials, volOps *r
 	return nil
 }
 
+func fillClusterIDFromMigrationReq(parameters map[string]string) (string, error) {
```

Comment:
this can cause issues if the radosNamespace configuration is present for this clusterID.

Reply:
this comment is not addressed and the function name can be `GetClusterIDFromMigrationVolume`

Reply:
shouldnt be an issue in this case.

Reply:
@humblec why it's not an issue can you please explain?

Reply:
@Madhu-1 radosnamespace wont be passed in this migration case.

Reply:
even if you don't pass the radosnamespace will be taken from the existing configmap and https://github.com/ceph/ceph-csi/blob/devel/internal/rbd/rbd_util.go#L1178-L1181 this is going to get called and things will fail and if you have multiple clusterID with the same mons again not sure things will work as each clusterID's can/might contain different radosNamespaces

Reply:
@humblec is this not addressed yet, function name need to be changed

Reply:
Done.